rotkehlchen/db/calendar.py

```diff
@@ -91,6 +94,53 @@ def deserialize_from_db(
         )
 
 
+class ReminderEntry(NamedTuple):
+    event_id: int
+    amount: int
```

Comment:
What are all these? You just need an event id and a timestamp or list of timestamps.